readthedocs/doc_builder/config.py

```diff
@@ -108,6 +106,15 @@ def formats(self):
             formats += ['pdf']
         return formats
 
+    @property
+    def build_image(self):
+        if self._project.container_image:
+            # Allow us to override per-project still
+            assert 'readthedocs/build' in self._project.container_image, (
+                'container image must be fully qualified')
+            return self._project.container_image
+        return 'readthedocs/build:{}'.format(self._yaml_config['build']['image'])
```

Comment:
Do we want the user to just put the last part?

Would it be interesting if I can put something like `humitos/rtd:latest` and install whatever I want in the docker image? Would it be too dangerous? Would it bring a lot of problems?

Just an idea (probably stupid, but...) / comment...